apps/console/src/features/private-key-jwt/pages/private-key-jwt-config-edit.tsx

```diff
@@ -0,0 +1,256 @@
+        setEnableTokenReuse(data.checked);
+        const updateData: any = {
+            operation: "ADD",
+            path: "/enableTokenReuse",
+            value: data.checked
+        };
+
+        updateJWTConfig(updateData).then(() => {
+            handleUpdateSuccess();
+        }).catch((error: any) => {
+            handleUpdateError(error);
+        });
+    };
+
+    /**
+     * This renders the enable toggle.
+     */
+    const connectorToggle = (): ReactElement => {
+        return (
+            <>
+                <Checkbox
+                    label={ enableTokenReuse ? "Token Reuse Enabled" : "Token Reuse Disabled" }
+                    toggle
+                    onChange={
+                        handleToggle
+                    }
+                    checked={ enableTokenReuse }
+                    readOnly={
+                        false
+                    }
+                    data-testId={ `${ componentId }-enable-toggle` }
+                />
+            </>
+        );
+    };
+
+
+    return (!isLoading ? (
+        <PageLayout
+            pageTitle="Private Key JWT Client Authentication for OIDC"
+            title={ t(
+                "console:manage.features.jwtPrivateKeyConfiguration.pageTitle"
+            ) }
+            description={ (
+                <>
+                    { t("console:manage.features.jwtPrivateKeyConfiguration.description" ) }
+                    <DocumentationLink
+                        link={ getLink("manage.validation.passwordValidation.learnMore") }
+                    >
+                        { t("common:learnMore") }
+                    </DocumentationLink>
+                </>
+            ) }
+            data-componentid={ `${ componentId }-page-layout` }
+            backButton={ {
+                "data-testid": `${ componentId }-page-back-button`,
+                onClick: handleBackButtonClick,
+                text: t(
+                    "console:manage.features.jwtPrivateKeyConfiguration.goBackToAccountSecurityConfig"
+                )
+            } }
+            bottomMargin={ false }
+            contentTopMargin={ true }
+            pageHeaderMaxWidth={ true }
+        >
+            {
+                connectorToggle()
+            }
+            {
+                <Grid className={ "mt-2" } >
+                    <Grid.Row columns={ 1 }>
+                        <Grid.Column width={ 10 }>
+                            <Message
+                                info
+                                content={ (
+                                    <>
+                                        <Icon name="info circle"/>
+                                        If enabled, the JTI in the JWT will be unique per the request if the previously
+                                        used JWT is not already expired. JTI (JWT ID) is a claim that provides a unique
+                                        identifier for the JWT.
```

Comment:
We need to i18n these strings, don't we?

Reply:
fixed with https://github.com/wso2/identity-apps/pull/3786/commits/dd8a9366531a26a1a44411371fe1db17b81925ae